### PR TITLE
Decompile object_gol (gohma actor)

### PR DESCRIPTION
**Pull request by Dragorn421**
(not the boss)

### CODE REVIEW

**Review by AngheloAlf — APPROVED** [review]

**Review by Roman971 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Goma/z_en_goma.h [hunk 1]
```diff
@@ -15,6 +15,34 @@ struct EnGoma;
...
+    /* 16 */ GOMA_LIMB_L_THIGH,
+    /* 17 */ GOMA_LIMB_R_LEG_ROOT1,
+    /* 18 */ GOMA_LIMB_R_LEG_ROOT2,
+    /* 19 */ GOMA_LIMB_R_SHIN_ROOT,
+    /* 20 */ GOMA_LIMB_R_FOOT_ROOT,
+    /* 21 */ GOMA_LIMB_R_FOOT,
+    /* 22 */ GOMA_LIMB_R_SHIN,
+    /* 23 */ GOMA_LIMB_R_THIGH,
+    /* 24 */ GOMA_LIMB_MAX
+} EnGe1Limb;
```
Comment (Roman971) [inline src/overlays/actors/ovl_En_Goma/z_en_goma.h:44]:
```suggestion
} EnGomaLimb;
```

**Review by Roman971 — CHANGES_REQUESTED** [review]
Nvm about the approval, I forgot to make a small comment

File: src/overlays/actors/ovl_En_Goma/z_en_goma.h (hunk 1 above)
Comment (fig02) [reply src/overlays/actors/ovl_En_Goma/z_en_goma.h:44]: gonna just commit this with github cause this is the only place the name needs changing

**Review by Roman971 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Goma/z_en_goma.h (hunk 1 above)
Comment (Dragorn421) [reply src/overlays/actors/ovl_En_Goma/z_en_goma.h]: oops lol

**Pull request merged**